src/OVAL/probes/independent/textfilecontent54.c

```diff
@@ -254,12 +260,13 @@ static int process_file(const char *path, const char *file, void *arg)
 	 * to return 'FTS_SL' and the presence of a valid target has to
 	 * be determined with stat().
 	 */
-	if (stat(whole_path, &st) == -1)
+	whole_path_with_prefix = oscap_sprintf("%s%s", prefix ? prefix : "", whole_path);
```

Comment:
I wish you declared the variable here and initialized it at the same time. We don't have to follow C89. I don't care enough to hold up this PR because of it though.